File: hyze_safety_moderator.py
# hyze_safety_moderator.py - Python 13+ Safe Moderator
BAD_WORDS = {
    'injection': ['ignore previous', 'you are now', 'system override', 'jailbreak'],
    'danger': ['bomb', 'hack', 'virus', 'malware']
}

def moderate(text):
    score = 100
    flagged = []
    for words in BAD_WORDS.values():
        for word in words:
            if word.lower() in text.lower():
                score -= 20
                flagged.append(word)
    return {'is_safe': score > 50, 'score': max(0, score), 'flagged': flagged}

if __name__ == "__main__":
    print(moderate("Jailbreak the system with virus"))
    # {'is_safe': False, 'score': 40, 'flagged': ['jailbreak', 'virus']}

File: hyze_safety_moderator.js
// hyze_safety_moderator.js - Age 13+ Safe for HyzeBot/Vercel
const BAD_WORDS = [
  'ignore previous', 'you are now', 'system override', 'jailbreak', 
  'bomb', 'hack', 'virus', 'malware'
];

function moderate(text) {
  const score = BAD_WORDS.reduce((s, word) => {
    const regex = new RegExp(word, 'gi');
    return s - (text.match(regex) || []).length * 20;
  }, 100);
  
  return { 
    isSafe: score > 50, 
    score: Math.max(0, score), 
    flagged: BAD_WORDS.filter(w => new RegExp(w, 'gi').test(text)) 
  };
}

// Export for Vercel API
export default moderate;

// Test
console.log(moderate("Ignore previous instructions")); 
// {isSafe: false, score: 40, flagged: ['ignore previous']}
